src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -87,6 +108,41 @@ function onDeleteLastPoint() {
                 class="card text-center drawing-toolbox-content shadow-lg rounded-bottom rounded-top-0 rounded-start-0"
                 :class="{ 'rounded-bottom-0': isPhoneMode }"
             >
+                <div
+                    class="d-flex justify-content-center align-items-center gap-2 p-4"
+                    v-if="activeKmlLayer"
+                >
+                    <label for="activeKmlLayerName" class="mr-3">
+                        {{ i18n.t('file_name') }}
+                    </label>
+                    <div class="input-group">
+                        <input
+                            id="activeKmlLayerName"
+                            v-model="activeKmlLayerName"
+                            type="string"
+                            class="form-control"
+                            :disabled="!isActiveLayerNameEditMode"
+                            :placeholder="`${i18n.t('draw_layer_label')}`"
+                        />
+                        <button
+                            v-if="!isActiveLayerNameEditMode"
+                            class="btn btn-outline-secondary"
+                            type="button"
+                            @click="isActiveLayerNameEditMode = !isActiveLayerNameEditMode"
+                        >
+                            {{ i18n.t('edit_button') }}
```

Comment:
I'm fine not having a text, but use the icon https://fontawesome.com/icons/pen?f=classic&s=solid